example/infrastructure/src/index.ts

```diff
@@ -156,36 +156,64 @@ const main = async () => {
     code: aws_lambda.Code.fromAsset("../../llrt-lambda-arm64.zip"),
     compatibleRuntimes: [
       aws_lambda.Runtime.NODEJS_16_X,
-      aws_lambda.Runtime.NODEJS_20_X,
+      aws_lambda.Runtime.NODEJS_18_X,
       aws_lambda.Runtime.NODEJS_20_X,
       aws_lambda.Runtime.NODEJS_LATEST,
-      aws_lambda.Runtime.PROVIDED_AL2,
+      aws_lambda.Runtime.PROVIDED_AL2023,
     ],
     compatibleArchitectures: [aws_lambda.Architecture.ARM_64],
   });
 
-  // LLRT hello
-  const helloLlrtFunction = new aws_lambda.Function(
+  // Amazon Linux runtime LLRT hello
+  const helloLlrtAlRuntimeFunction = new aws_lambda.Function(
     stack,
-    "HelloLlrtFunction",
+    "HelloLlrtAlRuntimeFunction",
     {
-      functionName: "example-hello-llrt",
+      functionName: "example-hello-llrt-al-runtime",
       code: aws_lambda.Code.fromAsset(sourceDirs["hello.mjs"]),
       ...props,
       environment: {},
-      runtime: aws_lambda.Runtime.PROVIDED_AL2,
+      runtime: aws_lambda.Runtime.PROVIDED_AL2023,
+      layers: [llrtLayer],
+    }
+  );
+
+  // Node runtime LLRT hello
+  const helloLlrtNodeRuntimeFunction = new aws_lambda.Function(
```

Comment:
I don't think we need to deploy this option. Maybe document it in README instead :)

Reply:
Yeah, I mentioned it in the issue:

> It doesn't seem that AWS_LAMBDA_EXEC_WRAPPER is well documented in the AWS docs, there's only a couple of mentions of that environment variable in examples. It might be a good idea to add it this project README, as well as shed some light on the advantage of using a Node runtime instead of AL (I assume different packages/libraries attached to the image?) I'll leave that up to you.

This example is not very visible, and doesn't explain why you would choose it over AL2023

Reply:
There is little to no value. Maybe best option here would be to add a comment around `compatibleRuntimes` explaining that EXEC_WRAPPER can be used. Or simply remove Node from `compatibleRuntimes`. What do you think?

Reply:
I'm very much removing them from the list of compatible runtimes if there is no point for it. We would list additional runtimes that require an env variable, which silently fails if it's not set (Node is used, not LLRT), and all for no meaningful gain. It doesn't feel like good DX, showing something that is just technically feasible

Reply:
Agree!

Reply:
Done 👍